qiskit/backends/basebackend.py

```diff
@@ -68,8 +68,13 @@ def properties(self):
 
     def status(self):
         """Return backend status"""
-        return AvailableToOperationalDict(
-            {'name': self.name(), 'operational': True, 'pending_jobs': 0})
+        return {'backend_name': self.name(),
+                #  Backend version in the form X.X.X
+                'backend_version': qiskit.__version__,
+                #  Backend operational and accepting jobs (true/false)
+                'operational': False,
```

Comment:
Low-importance: can you double-check if defaulting to `False` is intentional? It might have no real effect (specially for ibmq backends), but it might make  Aer backends "not operational" if they end up calling this method directly.

Reply:
I was originally thinking Aer backends would override the method when they were implemented so that status was explicit but maybe that's uneccesary. I'll change.